### DISCUSSION TITLE
this thing could've stolen everything from me just because im russian

### DISCUSSION THREAD

**Discussion by rech2020** [Tyrrrz/DiscordChatExporter#1270] [General]
i've run this thing and then this popup appeared
![image](https://github.com/user-attachments/assets/91a1cc40-7219-480d-98cc-a1933b08b151)
now i dont feel safe since this thing could've just stolen everything from me silently

**Comment by CanePlayz** [comment]
It cannot. The code is open source, everyone can check that there's nothing in the code that can do this.

**Comment by Tyrrrz** [comment]
> now i dont feel safe

Cry me a river

**Reply by DmitryYazovTNO1** [reply]
do you hate russians? or do you hate rashists only? im curious

**Reply by webknjaz** [reply]
What's the difference between passive and active nazis?

**Reply by DmitryYazovTNO1** [reply]
> What's the difference between passive and active nazis?

a rashist is a person who supports putin, not all russians are rashists, im a russian living in israel and i support ukraine.

**Reply by webknjaz** [reply]
That's not true. It's not one person that is bad, it's the society who elevated the person to represent them. The concept of ruscism spreads beyond that. It's inherently connected to colonialism and imperialist ideas. It's not attached to the color of the passport either. For example, so-called russian "opposition" is no different from the rest in their actions. Nor are the indifferent passive bystanders. This is because it's so deep in the culture that such behaviors are acceptable. The amount of r*ssians I met who are not also ruscists is below 0.01%, that's how many pass the dignity bar. But again, despite the high correlation with citizenship, I saw many with different passports and one thing in common — having been influenced by the toxic and rotten r*ssian "culture".
I think, my personal top test questions to determine with high accuracy if one is a ruscist or not, are asking about Crimea, the Tulsa provocation of 2003, genocides in Ichkeria and Georgia etc, what one did personally to stop the atrocities, watching how their reaction/behavior aligns with the propaganda and other incarnations of said "culture's" toxicity, like that cosplaying a "bigger brother" nonsense. The paradox of such a culture is that a huge chunk of people don't even have tools to realize how toxic and imperialist it is.

So no, p*tin is an example of ruscism but not its entirety. The culture that supports his privilege is.

**Reply by DmitryYazovTNO1** [reply]
so you're viewing russians as uncultured swines?

**Reply by webknjaz** [reply]
That's a manipulation, I didn't say that. They come in different shapes and sizes but what they call culture is usually either stolen due to colonialism or hostile/offensive to anybody who dares to have a different identity.

Self-education can be started here: https://istpublishing.org/en/russian-colonialism-101-maksym-eristavi

**Reply by daniel-ramrain** [reply]
@webknjaz А Израиль и Иран? США? Афганистан? Англия? Это другое или недостаточно потужно?

**Reply by webknjaz** [reply]
@daniel-ramrain go away, ruscist nazi 🤡 🤡 🤡

**Comment by MaxGraey** [comment]
> now i dont feel safe

Man, you literally live in russia! Feeling unsafe should be a normal for you =)

**Comment by webknjaz** [comment]
> now i dont feel safe
> *just* because im russian

Dude, sounds like you have a more serious problem than conspiracy theories around fully open source software 🤔

**Comment by superintendent2521** [comment]
what the fuck is with these comments here, this person has done nothing? they live in a country that has an active war with a country for no reason, and yet you are still making fun of him? why? what did they do?

**Reply by DmitryYazovTNO1** [reply]
literally almost every ukrainian programmer try not to put anti russian war protestware bloat in their programs (IMPOSSIBLE CHALLENGE), (i know russia has done awful shit to ukraine but that dosent mean the programmers living in ukraine have to put protestware in all of their programs, if you want a version of DiscordChatExporter without the political bloat, use this fork by nulldg: https://github.com/nulldg/DiscordChatExporterPlus

**Reply by CanePlayz** [reply]
Is the bloat in the room with us right now?

**Reply by webknjaz** [reply]
@DmitryYazovTNO1 wait, you're claiming to support Ukraine but at the same time you're advocating for suppressing the Ukrainian voices by silencing whatever information they share. Which one is it, then? They are the opposite. Anyway, actions speak louder than words.

**Reply by webknjaz** [reply]
@superintendent2521 that person did nothing right :man_shrugging: Having a slave mindset doesn't excuse them.

**Reply by DmitryYazovTNO1** [reply]
@webknjaz no im not??? i support ukraine, im only just tired of seeing protestware bloat, we all know that russia is doing horrible shit to your country, you dont need to put it everywhere, you dont need to restrict MILLIONS of people from using a program for living in certain countries, this is just hilarious.

**Reply by webknjaz** [reply]
LOL

**Reply by webknjaz** [reply]
You don't really know what support is, then.

**Reply by daniel-ramrain** [reply]
Если написать "хуй" на заборе - это настоящая поддержка, то мир окончательно сошёл с ума)))
